services/web/client/source/class/osparc/desktop/StudyEditor.js

```diff
@@ -111,7 +111,7 @@ qx.Class.define("osparc.desktop.StudyEditor", {
               })
               .catch(err => {
                 if ("status" in err && err["status"] == 423) { // Locked
-                  const msg = study.getName() + this.tr(" is already opened");
+                  const msg = study.getName() + this.tr(" was already opened");
```

Comment:
```suggestion
                  const msg = study.getName() + this.tr(" is already opened");
```

it is a fact isn't it?

Reply:
right